libs/solid/skeleton/src/lib/components/base-layout/base-layout.component.html

```diff
@@ -9,6 +9,12 @@
       <!-- <div class="left-border">
         <span>Glossar</span>
       </div> -->
+      <div class="logo">
+        <img
+          src="assets/info/Hochschule_Logo.svg"
```

Comment:
Der Pfad zum Logo sollte nicht hardcoded sein.
Mein Vorschlag: In solid-core-config wird der Dateiname (oder der ganze Pfad) definiert:
https://github.com/zentrumnawi/solid-frontend/blob/an/logo-in-toolbar/apps/planty/src/app/solid-core-config.ts
Dazu muss auch die Definition von Solid-Core angepasst werden:
https://github.com/zentrumnawi/solid-frontend/blob/an/logo-in-toolbar/libs/solid/core/src/lib/solid-core-config.ts

Die neuen Felder könnten `appLogoSmall` und `appLogoLarge` heißen und optional sein (müsste dann `appLogosmall?` sein, oder?)